alignak/scheduler.py

```diff
@@ -442,26 +540,39 @@ def add_notification(self, notif):
         :type notif: alignak.notification.Notification
         :return: None
         """
+        if notif.uuid in self.actions:
+            logger.debug("Already existing notification: %s", notif)
+            return
+
         self.actions[notif.uuid] = notif
+        self.nb_notifications += 1
+
         # A notification ask for a brok
         if notif.contact is not None:
             brok = notif.get_initial_status_brok()
             self.add(brok)
 
     def add_check(self, check):
-        """Add a check into checks list
+        """Add a check into the scheduler checks list
 
         :param check: check to add
         :type check: alignak.check.Check
         :return: None
         """
         if check is None:
             return
+        if check.uuid in self.checks:
+            logger.debug("Already existing check: %s", check)
```

Comment:
If we have it, it's not normal, so prefer use error or warning instead debug.
Same for actions, event_handler...